app/view/Client/frmClient.cs

```diff
@@ -67,15 +75,23 @@ private void btnAddPatient_Click(object sender, EventArgs e)
             using (frmClientPatientModal newClientPatientForm = new frmClientPatientModal())
```

Comment:
Add a validation that restricts the user to add a Patient/Pet, when there's no loaded client.